pubtools/_quay/tag_images.py

```diff
@@ -101,56 +101,162 @@
         "help": "UMB topic to send the message to.",
         "required": False,
         "type": str,
+        "default": "VirtualTopic.eng.pub.quay_tag_image",
     },
 }
 
 
-def tag_images(args):
-    """Tag images main function."""
+def construct_kwargs(args):
+    """
+    Construct a kwargs dictionary based on the entered command line arguments.
+    Args:
+        args (argparse.Namespace):
+            Parsed command line arguments.
+    Returns (dict):
+        Keyword arguments for the 'untag_images' function.
+    """
+    dest_refs = args.dest_ref if isinstance(args.dest_ref, list) else [args.dest_ref]
+    kwargs = {"source_ref": args.source_ref, "dest_refs": dest_refs}
+    if args.all_arch:
```

Comment:
How about something like this?
```
some_list = ['all_arch', 'quay_user']
for key in some_list:
    value = getattr(args,key, None)
    if value:
        kwargs[key] = value
 ```
Apologies for terrible naming skills

Reply:
Yeah, the current approach was pretty dumb. I tried to improve it (also for untag entrypoint).